snap/snapcraft.yaml

```diff
@@ -332,40 +332,6 @@ parts:
   ccache:
```

Comment:
I'm confused: if this removes `ccache`, why is this bit still here? And how does the snap know to use ccache when the dependency isn't listed anywhere?

Reply:
Refer [Remote (reusable) parts - doc - snapcraft.io](https://forum.snapcraft.io/t/remote-reusable-parts/4233) for more info about remote parts.  Snapcraft maintains a regularly updated cache of remote parts.